woocommerce/woocommerce
# Remove Codisto from default free extensions.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

In this PR, we remove Codisto (aka. "Amazon, eBay & Walmart Integration for WooCommerce") from the default free extensions for onboarding wizard and task list. Default free extensions is used when users disable suggestions from WooCommerce.com website in WooCommerce settings.

We also remove Codisto from one of our tests.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to WooCommerce Settings > Advanced > Woo.com page (`/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`), and turn off "Show Suggestions" option. See screenshot below.

![image](https://github.com/woocommerce/woocommerce/assets/417342/70a838bc-29d4-4b56-a1cb-f66d993ca663)

2. Go to WooCommerce Home > Get more sales page (`/wp-admin/admin.php?page=wc-admin&task=marketing`). "Codisto for WooCommerce" should not be in the "Recommended marketing extensions" card. See screenshot below.

![image](https://github.com/woocommerce/woocommerce/assets/417342/6eef1c06-6364-4624-a278-8b73a00938e3)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Remove Codisto from the default free extensions for onboarding wizard and task list.

#### Comment 

</details>
